Source/SLKTextView.m

```diff
@@ -138,7 +138,6 @@ - (UILabel *)placeholderLabel
         _placeholderLabel = [UILabel new];
         _placeholderLabel.clipsToBounds = NO;
         _placeholderLabel.autoresizesSubviews = NO;
-        _placeholderLabel.numberOfLines = 1;
```

Comment:
@jedmund Sup yo?, Sweet PR, you should keep this line though so that default number of lines is 1 as per the documentation you wrote.